docs/topics/multiplatform/multiplatform-publish-lib.md

```diff
@@ -50,24 +50,30 @@ in the publication's scope.
 ## Host requirements
 
 Kotlin/Native supports cross-compilation, allowing any host to produce necessary `.klib` artifacts.
+However, there are still some specifics you should keep in mind.
+
+### Compilation for Apple targets
+<primary-label ref="experimental-opt-in"/>
 
 To produce artifacts for projects with Apple targets, you'd normally need an Apple machine.
-However, if you want to use other hosts, you can set this [Experimental](components-stability.md#stability-levels-explained)
-option in your `gradle.properties` file:
+However, if you want to use other hosts, set this option in your `gradle.properties` file:
 
 ```none
 kotlin.native.enableKlibsCrossCompilation=true
 ```
 
-> To build [final binaries](multiplatform-build-native-binaries.md) for Apple targets, you still need to use a Mac machine.
->
-{style="note"}
+Cross-compilation is currently Experimental and has some limitations. You still need to use a Mac machine if:
+
+* Your library has a [cinterop dependency](native-c-interop.md).
```

Comment:
I'm not sure if it will be obvious that CocoaPods integration is also affected since it is based on the cinterop.

Reply:
Thank you! added as a separate item